web/src/beta/features/ProjectSettings/innerPages/PublicSettings/PublicSettingsDetail.tsx

```diff
@@ -1,6 +1,6 @@
 import { IMAGE_TYPES } from "@reearth/beta/features/AssetsManager/constants";
 import { Button, Collapse } from "@reearth/beta/lib/reearth-ui";
-import defaultBetaProjectImage from "@reearth/beta/ui/assets/defaultBetaProjectImage.png";
+import defaultProjectBackgroundImage from "@reearth/beta/ui/assets/defaultProjectBackgroundImage.png";
```

Comment:
Please check the assets, i think png is okey, but there's only svg file provided at the path.
We want to use more svg but only when the image is vector graphic. In this case the background image is actually pixel graphic, it will be embed so doesn't help when saving as svg.
BTW it's also okey for now to have the dark gray background included in this image, but might be better to have a transparent background, we can more easily change the background color or apply light mode with this in the future.

Reply:
Thanks for spotting this. Yes, in the future we might want to be able to have a light background.